assets/script.js

```diff
@@ -57,6 +57,8 @@ var submitButton = document.querySelector(".submit-button");
 var timeLeft = 40;
 var scoreboardPage = document.querySelector(".scoreboard-page");
 var replayButton = document.querySelector(".replay");
+var currentQuestion;
+var timeInterval;
```

Comment:
the bugs you were running into were partially caused by these values not being global. This made it harder to change these values in different places in the code.